catalog/ui/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/query/utility/CqlQueryResponse.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.utility;
+
+import ddf.catalog.operation.QueryResponse;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public interface CqlQueryResponse {
```

Comment:
we might want to mark the interfaces as experimental 
https://github.com/codice/ddf/blob/b054ac205251e31390e5ccc9830dfe310a3f2659/catalog/core/catalog-core-attachment/src/main/java/org/codice/ddf/attachment/AttachmentInfo.java#L18